docker-compose.yml

```
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+  safaribooks:
+    build: .
+    volumes:
+       - ./:/safaribooks
```

With suggested Dockerfile changes, this volume mount can be just for the `Books` subfolder.
```suggestion
       - ./Books:/safaribooks/Books
```
Or
```suggestion
       - ${PWD}/Books:/safaribooks/Books
```

---

When we keep the entire project in a shared place we don't need to add this.